### PR TITLE
feat(workspace): workspace_desktop support update enterprise_project_id

### PR DESCRIPTION
**Pull request by wangyuancheng01**
**What this PR does / why we need it**: workspace_desktop support update enterprise_project_id

**Which issue this PR fixes**:
*(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*
fixes #xxx

**Special notes for your reviewer**:

**Release note**:

```release-note

```

## PR Checklist

* [x] Tests added/passed.
* [x] Documentation updated.
* [x] Schema updated.

## Acceptance Steps Performed

```
make testacc TEST="./huaweicloud/services/acceptance/workspace" TESTARGS="-run TestAccDesktop_UpdateWithEpsId"
...
==> Checking that code complies with gofmt requirements...
TF_ACC=1 go test ./huaweicloud/services/acceptance/workspace -v -run TestAccDesktop_UpdateWithEpsId -timeout 360m -parallel 4
=== RUN   TestAccDesktop_UpdateWithEpsId
=== PAUSE TestAccDesktop_UpdateWithEpsId
=== CONT  TestAccDesktop_UpdateWithEpsId
--- PASS: TestAccDesktop_UpdateWithEpsId (1287.16s)
PASS
ok      github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/services/acceptance/workspace 1287.195s
```

### CODE REVIEW

File: huaweicloud/services/acceptance/workspace/resource_huaweicloud_workspace_desktop_test.go [hunk 1]
```diff
@@ -102,6 +102,74 @@ func TestAccDesktop_basic(t *testing.T) {
...
+	rc := acceptance.InitResourceCheck(
+		resourceName,
+		&desktop,
+		getDesktopFunc,
+	)
+
+	resource.ParallelTest(t, resource.TestCase{
+		PreCheck: func() {
+			acceptance.TestAccPreCheck(t)
+			acceptance.TestAccPreCheckEpsID(t)
```
Comment (Jason-Zhang9309) [inline huaweicloud/services/acceptance/workspace/resource_huaweicloud_workspace_desktop_test.go]: remove this precheck
```suggestion
```

Comment (wangyuancheng01) [reply huaweicloud/services/acceptance/workspace/resource_huaweicloud_workspace_desktop_test.go]: down

File: huaweicloud/services/acceptance/workspace/resource_huaweicloud_workspace_desktop_test.go [hunk 2]
```diff
@@ -228,6 +285,58 @@ resource "huaweicloud_workspace_desktop" "test" {
 `, testAccDesktop_base(rName), rName)
 }
 
+func testAccDesktop_withEPSId(rName, epsId string) string {
+	return fmt.Sprintf(`
+%[1]s
+
+locals {
+  data_volume_sizes = [50, 90, 20, 40]
```
Comment (ShiChangkuo) [inline huaweicloud/services/acceptance/workspace/resource_huaweicloud_workspace_desktop_test.go]: I think we can not care the data volumes in this acceptance test, please check it.

Comment (wangyuancheng01) [reply huaweicloud/services/acceptance/workspace/resource_huaweicloud_workspace_desktop_test.go]: down

**Pull request merged**